### PR TITLE
[#1905][FOLLOWUP] docs: Add documents after introducing XXX_JAVA_OPTS env var

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes were proposed in this pull request?

Add some necessary docs.

### Why are the changes needed?

Fix: #1905

### Does this PR introduce _any_ user-facing change?

Yeah, docs page.

### How was this patch tested?

No need

### CODE REVIEW

File: docs/server_guide.md [hunk 1]
```diff
@@ -174,6 +174,14 @@ XMX_SIZE=40g
 MAX_DIRECT_MEMORY_SIZE=360g
 ```
 
+Besides, you can add extra jvm arguments for shuffle server by specific `UNIFFLE_SHUFFLE_SERVER_JAVA_OPTS` in `rss-env.sh`
+
+Something like:
```
Comment (rickyma) [inline docs/server_guide.md:179]: `e.g.` or `For example:` will be better

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -100,7 +100,6 @@ jobs:
         **/target/surefire-reports/*.xml
 
   docker:
-    needs: [integration]
```
Comment (EnricoMi) [inline .github/workflows/build.yml:103]: I thin you need to rebase with / merge latest master.

File: docs/coordinator_guide.md [hunk 3]
```diff
@@ -36,6 +36,11 @@ This document will introduce how to deploy Uniffle coordinators.
      HADOOP_HOME=<hadoop home>
      XMX_SIZE="16g"
    ```
+   Besides, you can add extra jvm arguments for shuffle server by specific `UNIFFLE_COORDINATOR_JAVA_OPTS` in `rss-env.sh`
+   Something like:
```
Comment (EnricoMi) [inline docs/coordinator_guide.md:40]:
```suggestion
   You can add extra JVM arguments for the Uniffle coordinator by specifying `UNIFFLE_COORDINATOR_JAVA_OPTS` in `rss-env.sh`.
   Example:
```

File: docs/dashboard_guide.md [hunk 4]
```diff
@@ -32,6 +32,13 @@ rss.dashboard.http.port 19997
 coordinator.web.address http://coordinator.hostname:19998/
 ```
 
+### Configure related JVM parameters
+you can add extra jvm arguments for shuffle server by specific `UNIFFLE_DASHBOARD_JAVA_OPTS` in `RSS_HOME/bin/rss-env.sh`
+Something like:
```
Comment (EnricoMi) [inline docs/dashboard_guide.md:37]:
```suggestion
You can add extra JVM arguments for the Uniffle dashboard by specifying `UNIFFLE_DASHBOARD_JAVA_OPTS` in `RSS_HOME/bin/rss-env.sh`
Example:
```

File: docs/server_guide.md (hunk 1 above)
Comment (EnricoMi) [inline docs/server_guide.md:179]:
```suggestion
You can add extra JVM arguments for the Uniffle shuffle server by specifying `UNIFFLE_SHUFFLE_SERVER_JAVA_OPTS` in `rss-env.sh`

Example:
```

**Review by EnricoMi — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (maobaolong) [reply .github/workflows/build.yml:103]: done

**Comment by jerqi** [comment]
You push the code forcely and revert the suggestions. So I mark the suggestions unresolved again.

File: docs/server_guide.md (hunk 1 above)
Comment (maobaolong) [reply docs/server_guide.md]: done

**Comment by maobaolong** [comment]
@jerqi Sorry, I accidentally force pushed commit yesterday, I accept the suggestion commits from @EnricoMi  again. PTAL

**Comment by maobaolong** [comment]
ping @jerqi @EnricoMi

**Review by jerqi — APPROVED** [review]
LGTM. Wait for @EnricoMi for 2 days.

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Comment by rickyma** [comment]
Merged. Thanks @maobaolong @EnricoMi @jerqi

**Pull request merged**